NuGet.Config

```diff
@@ -54,12 +59,102 @@
       <package pattern="envdte80" />
       <package pattern="envdte90" />
       <package pattern="envdte90a" />
+      <package pattern="FluentAssertions" />
+      <package pattern="ILMerge" />
+      <package pattern="Lucene.Net" />
+      <package pattern="MessagePack" />
+      <package pattern="Microsoft.AspNetCore.App.Ref" />
+      <package pattern="Microsoft.Build" />
+      <package pattern="Microsoft.Build.*" />
+      <package pattern="Microsoft.Build.Artifacts" />
+      <package pattern="Microsoft.Build.Framework" />
+      <package pattern="Microsoft.Build.Locator" />
+      <package pattern="Microsoft.Build.Runtime" />
+      <package pattern="Microsoft.Build.Tasks.Core" />
+      <package pattern="Microsoft.Build.Utilities.Core" />
+      <package pattern="Microsoft.CodeAnalysis" />
+      <package pattern="Microsoft.CodeAnalysis.Analyzers" />
+      <package pattern="Microsoft.CodeAnalysis.AnalyzerUtilities" />
+      <package pattern="Microsoft.CodeAnalysis.BannedApiAnalyzers" />
+      <package pattern="Microsoft.CodeAnalysis.Common" />
+      <package pattern="Microsoft.CodeAnalysis.CSharp" />
+      <package pattern="Microsoft.CodeAnalysis.CSharp.Workspaces" />
+      <package pattern="Microsoft.CodeAnalysis.EditorFeatures.Text" />
+      <package pattern="Microsoft.CodeAnalysis.Features" />
+      <package pattern="Microsoft.CodeAnalysis.PublicApiAnalyzers" />
+      <package pattern="Microsoft.CodeAnalysis.Scripting.Common" />
+      <package pattern="Microsoft.CodeAnalysis.VisualBasic.Workspaces" />
+      <package pattern="Microsoft.CodeAnalysis.Workspaces.Common" />
```

Comment:
When I had an attempt myself to upgrade to the 17.10 VS SDK, I found there was at least one or two code analysis packages that needed to come from the vside feed, but my memory is that most did not.

Some of these, the BannedApiAnalyzers and PublicApiAnalyzers, just feel infeasible that there are vside special versions, and I'm confident that we're using nuget.org versions available on dotnet-public.